conf/SimpleConfiguration.php

```diff
@@ -0,0 +1,268 @@
+						}
+					}
+
+					if(isset($config["check"][2])) {
+						if(preg_match("/{(.*)}/si", $config["check"][2], $matches)) {
+							$val2 = self::getVariableByAlias($matches[2]);
+						}
+						else {
+							$val2 = $config["check"][2];
+						}
+					}
+
+					switch($config["check"][0])
+					{
+						case "=":
+							if($val1 == $val2) {
+								return $this->parseDynamicConfigs($config["true"]);
+							}
+							else {
+								return $this->parseDynamicConfigs($config["false"]);
+							}
+							break;
+						case "<>":
+							if($val1 != $val2) {
+								return $this->parseDynamicConfigs($config["true"]);
+							}
+							else {
+								return $this->parseDynamicConfigs($config["false"]);
+							}
+							break;
+						case "<":
+							if($val1 < $val2) {
+								return $this->parseDynamicConfigs($config["true"]);
+							}
+							else {
+								return $this->parseDynamicConfigs($config["false"]);
+							}
+							break;
+						case ">":
+							if($val1 > $val2) {
+								return $this->parseDynamicConfigs($config["true"]);
+							}
+							else {
+								return $this->parseDynamicConfigs($config["false"]);
+							}
+							break;
+						case "<=":
+							if($val1 <= $val2) {
+								return $this->parseDynamicConfigs($config["true"]);
+							}
+							else {
+								return $this->parseDynamicConfigs($config["false"]);
+							}
+							break;
+						case ">=":
+							if($val1 >= $val2) {
+								return $this->parseDynamicConfigs($config["true"]);
+							}
+							else {
+								return $this->parseDynamicConfigs($config["false"]);
+							}
+							break;
+					}
+				}
+				else {
+					foreach($config as $portion) {
+						if(is_array($portion)) {
+							$val .= $this->parseDynamicConfigs($portion, true);
+						}
+						else {
+							if($resolve) {
+								$portionVal = self::getVariableByAlias($portion);
+								if(is_array($portionVal)) {
+									$portionVal = $this->parseDynamicConfigs($portionVal);
+								}
+								$val .= $portionVal;
+							} else {
+								$val .= $portion;
+							}
+						}
+					}
+					return $val;
+				}
+			}
+			else {
+				return $this->parseDynamicConfigs(self::getVariableByAlias($config));
+			}
+		}
+	}
+
+	public static function reload($config_directory = null)
+	{
+		$configs = self::instance();
+		if(is_null($config_directory)) {
+			$config_directory = $configs->config_directory;
+		}
+
+		return self::instance($config_directory, true);
+	}
+
+	public static function getVariableByAlias($alias)
+	{
+		$expanded = explode(".", $alias);
+		$val = null;
+		switch($expanded[0])
+		{
+			case "this":
+				$val = self::$instance;
+				break;
+			case "server":
+				$val = $_SERVER;
+				break;
+			case "get":
+				$val = $_GET;
+				break;
+			case "post":
+				$val = $_POST;
+				break;
+			case "session":
+				$val = $_SESSION;
+				break;
+			default:
+				return null;
+		}
+
+		for($i = 1; $i < count($expanded); $i++) {
+			$val = $val[$expanded[$i]];
```

Comment:
We should really check that the index exists to avoid flooding the log with notices.